pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1945,6 +1946,27 @@ func handleDelete(ctx *domainContext, key string, status *types.DomainStatus) {
 		status.UUIDandVersion, status.DisplayName)
 }
 
+// lookForRktRunErrors checks for xen errors or any other error which
+// are encountered while rkt run
+// rkt run returns two standard outputs when rkt runs successfully
+// so we need to remove them while checking for errors
+func lookForRktRunErrors(str string) error {
+	var output []string
+	standardOutput1 := "run: group \"rkt\" not found, will use default gid when rendering images"
+	standardOutput2 := "Parsing config from /var/lib/rkt/pods/run/"
+	strSlice := strings.Split(str, "\n")
+	for _, v := range strSlice {
+		if v != standardOutput1 && !strings.HasPrefix(v, standardOutput2) {
```

Comment:
So basically, are you just filtering these two lines from the rkt output and printing it out as errors.. is it?

Reply:
No, we are filtering out these two lines and the rest we are declaring an error.

Reply:
That's wrong. Errors should be on stderr; should be no need to parse stdout (unless rkt is utterly broken and doesn't know the difference between stderr and stdout)

Reply:
Container stderr is appearing in rkt run stdout.